File: programme/number-to-string/number-to-string.js
// Write a programmeto convert number to strings

const words = [
  "Zero",
  "One",
  "Two",
  "Three",
  "Four",
  "Five",
  "Six",
  "Seven",
  "Eight",
  "Nine",
];

convert(12223455);

function convert(input) {
  if (typeof input != "number") return console.error("Not a number");
  console.log("Input  :", input);

  var inp = input.toString();

  let arr = [];

  for (let i = 0; i < inp.length; i++) {
    if (inp[i] === ".") arr.push("point");
    else arr.push(words[inp[i]]);
  }

  console.log("Output :", ...arr);
}

File: programme/number-to-string/number-to-string.cpp
// Write a programmeto convert number to strings
#include <iostream>
#include <string>

using namespace std;

int main()
{
    int a, n = 0;
    string word[] = {"Zero", "One", "Two", "Three", "Four", "Five", "Six", "Seven", "Eight", "Nine"};
    cout << "Input  : ";
    cin >> a;
    string op = "";
    while (a > 0)
    {
        n = a % 10;
        op.insert(0, word[n]);
        op.insert(0, " ");
        a = a / 10;
    }
    cout << "Output :" << op << endl;
    return 0;
}
